### DISCUSSION TITLE
posts/2023/09/given-in-life/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#22] [General]
# posts/2023/09/given-in-life/

卖猪肚鸡的阿姨今天给我送了一瓶饮料。虽然是因为饮料上的吸管脱落了，阿姨说别的同学会嫌弃，不想要。

https://guozheng.rbind.io/posts/2023/09/given-in-life/

**Comment by DylanYu233** [comment]
退出广东籍吧，自觉点🤣👉🏻

**Reply by residualsun1** [reply]
🤦‍♂️
